src/test/java/resourcesTests/PersonResourcesTest.java

```diff
@@ -0,0 +1,81 @@
+package resourcesTests;
```

Comment:
class should be named PersonsResourceTest (just like the resource should be named PersonsResource) - therefore the package should be resourceTests